PowerUp/app/src/main/res/values-sw600dp/dimens.xml

```diff
@@ -38,5 +38,9 @@
     <dimen name="avatar_room_width">120dp</dimen>
     <dimen name="avatar_room_height">120dp</dimen>
     <dimen name="avatar_room_continue_button_land">80dp</dimen>
+	<dimen name="completed_game_textview">80dp</dimen>
+    <dimen name="completed_game_width">300dp</dimen>
+    <dimen name="completed_game_height">220dp</dimen>
+    <dimen name="game_text_size">30dp</dimen>
```

Comment:
Changing the game text size is increasing the text size everywhere, including in the conversations. You should define a new variable for the text of these final screens and redefine it depending on the size of the screen, but not redefine the text size for the entire game